docs/UserGuide.md

```diff
@@ -216,15 +216,23 @@ Result for `fc Investment Plan`:
 
 Deletes the specified person from ClientHub.
 
-Format: `delete INDEX`
+Format: `delete NAME` or `d NAME` or `delete NAME/`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* Deletes the person with specified NAME
+* / is used to indicate specific name to delete
+    * For eg. if 2 contacts have names such as "David Li" and "David Lim", typing `delete David Li/` will delete the contact with the name "David Li".
+    * Name written before / must be **EXACT** name of the contact to be deleted.
+    * Order matters when using / to delete a contact.
+
+A **valid** `NAME` for delete should:
+    * Not be empty. 
+        * For eg. Just typing `delete` without providing any `NAME` will throw an error.
+    * Be a valid name that exists in the list of contacts.
+        * For eg. Typing `delete John Doe` when there is no contact with the name `John Doe` will throw an error.
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in ClientHub.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `delete John Doe` deletes the person named `John Doe`
+* `delete John Doe/` deletes the person named `John Doe` and not `John Doey`
```

Comment:
Well written User guide for the `delete` command. Maybe can add a picture of an example output of an example command.